alert-anomaly-risk/jsonld-contexts/alert-anomaly-risk.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "Alert": "https://uri.fiware.org/ns/data-models#Alert",
```

Comment:
avoid using the `uri.fiware.org` namespace (it was used in the past and is now deprecated). you can instead use `smartdatamodels.org` (like in models coming from SDM)